src/verilog/verilog_typecheck_expr.cpp

```diff
@@ -488,6 +489,31 @@ exprt verilog_typecheck_exprt::convert_expr_function_call(
 
 /*******************************************************************\
 
+Function: verilog_typecheck_exprt::isunknown
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+constant_exprt verilog_typecheck_exprt::isunknown(const constant_exprt &expr)
+{
+  // constant folding for $isunknown
+  auto bval = ::bval(expr);
+  auto bval_simplified = simplify_expr(bval, ns);
```

Comment:
Should this possibly use `verilog_simplifier`?

Reply:
Done